sdk/sync/go.mod

```diff
@@ -4,13 +4,17 @@ go 1.14
 
 require (
 	github.com/go-redis/redis/v7 v7.2.0
-	github.com/ipfs/testground v0.1.0
-	github.com/ipfs/testground/sdk/runtime v0.3.0
-	github.com/libp2p/go-libp2p-core v0.2.3
-	github.com/libp2p/go-openssl v0.0.4 // indirect
-	github.com/multiformats/go-multiaddr v0.1.1
+	github.com/ipfs/testground v0.4.0
```

Comment:
Mind running a gomod tidy here?

Reply:
this is after `go mod tidy`.

Reply:
I believe we have it in CI, so if `go mod tidy` produces something else, CI fails.